R/synthdid.R

```diff
@@ -96,11 +100,11 @@ sc_estimate = function(Y, N0, T0, X=array(dim=c(dim(Y),0)),
                        zeta.lambda=0, zeta.omega=sd(apply(Y,1,diff)),
                        lambda.intercept=FALSE, omega.intercept=FALSE,
                        weights = list(lambda=rep(0,T0), omega=NULL, vals=NULL),
-		       min.decrease=1e-3, max.iter=1e4) {
+		       min.decrease=1e-3*sd(apply(Y,1,diff)), max.iter=1e4) {
     synthdid_estimate(Y, N0, T0, X=X,
                        zeta.lambda=zeta.lambda, zeta.omega=zeta.omega,
                        lambda.intercept=lambda.intercept, omega.intercept=omega.intercept,
-                       weights = weights, min.decrease=1e-3, max.iter=1e4)
+                       weights = weights, min.decrease=min.decrease, max.iter=max.iter)
 }
```

Comment:
Pls remember to commit relevant autogenerated files too, here the function signature is changed to if you run roxygen it will produce a new doc file

![Screen Shot 2020-11-24 at 13 10 09](https://user-images.githubusercontent.com/7185264/100151740-6f0b2a80-2e56-11eb-8936-c9b8c35cb311.png)

Reply:
done